subprojects/docs/src/docs/userguide/dep-man/06-publishing/publishing_maven.adoc

```diff
@@ -256,6 +256,11 @@ The result is that the following artifacts will be published:
 
 The <<signing_plugin.adoc#signing_plugin, Signing Plugin>> is used to generate a signature file for each artifact. In addition, checksum files will be generated for all artifacts and signature files.
 
+[TIP]
+====
+`publishToMavenLocal` do not create checksum files to _$USER_HOME/.m2/repository_ since it works in Maven itself. If you want to validate locally, please validate checksum files `<project directory>/build` directory, which `publish` task generates.
```

Comment:
```suggestion
`publishToMavenLocal` does not create checksum files in `$USER_HOME/.m2/repository`.
Instead, `publishToMavenLocal` creates checksum files in the `<project directory>/build` directory during the `publish` task.
```

Suggestion: trim language a bit, make the solution to this issue easier to understand.

Reply:
Thanks a lot!!